### PR TITLE
Tricks page phase4

### PR DESCRIPTION
**Pull request by tanguyguillo**
asking PR for last rdv with Paolo @castroCrea

### CODE REVIEW

File: public/js/else.js [hunk 1]
```diff
@@ -1,23 +1,55 @@
 $(document).ready(function () {
-    var id
+    let id
+    let idPicture
 
+    // document.getElementById("comments").reset();
```
Comment (castroCrea) [inline public/js/else.js:5]: remove comments

File: public/js/else.js [hunk 2]
```diff
@@ -1,23 +1,55 @@
...
+    // img.src = "assets/img/background/empty.png"
+
+    // fonctionnal
+    // $('.collapseMain').click(function () {
+    //     console.log('click collapseMain')
+    //     var anchor = $(this).attr("href");
+    //     $("html, body").animate({
+    //         scrollTop: $(anchor).offset().top
+    //     }, "slow");
+    // });
```
Comment (castroCrea) [inline public/js/else.js:33]: remove comments if not used

File: public/js/else.js [hunk 3]
```diff
@@ -27,32 +59,80 @@ $(document).ready(function () {
                 console.log(response);
                 $('#deleteThisTrick').modal('toggle');
                 $('.' + id).addClass('displayNone');
+
+                // var visual = "<img src=\"assets/img/tricks/.jpg\empty.png />";
+                // $(visual).replaceAll(".empty");
```
Comment (castroCrea) [inline public/js/else.js:64]: remove comments

File: public/js/else.js [hunk 4]
```diff
@@ -27,32 +59,80 @@ $(document).ready(function () {
...
+    // $.fn.missingImg = function (options) {
+    //     var config = {
+    //         'source': 'assets/img/tricks/empty.png',
+    //         'alt': 'Image not found'
+    //     };
+    //     if (options) { $.extend(config, options); }
+    //     $(this).each(function () {
+    //         var $this = $(this),
+    //             oImg = this, // original DOM element
+    //             testImg = new Image(); // new DOM element
```
Comment (castroCrea) [inline public/js/else.js:84]: remove comments, check all code

File: src/Controller/TrickController.php [hunk 5]
```diff
@@ -3,340 +3,478 @@
...
+        return $this->render('tricks/details.html.twig', compact('trick', 'author', 'additionalPictures', 'Image', 'date', 'formComment', 'currentComments', 'userId'));  // 
     }
 
     /**
-     * function deleteFromDetail
+     * Function update (write)
      */
-    #[Route('/delete-tricks_from_detail/{id}', name: 'app_tricks_delete_from_detail', methods: ['Post'])]
-    public function deleteFromDetail(Request $request, Tricks $trick, TricksRepository $tricksRepository)
+    #[Route('/details/modifications/{slug}', name: 'modifications')]
```
Comment (castroCrea) [inline src/Controller/TrickController.php:112]: Slit the controller in smaller controller

**Review by castroCrea — COMMENTED** [review]
run prettier

File: public/js/else.js (hunk 1 above)
Comment (tanguyguillo) [reply public/js/else.js]: done

File: public/js/else.js (hunk 2 above)
Comment (tanguyguillo) [reply public/js/else.js]: done

File: public/js/else.js (hunk 4 above)
Comment (tanguyguillo) [reply public/js/else.js]: done

**Pull request merged**